app/views/housing_attributes/_edit.haml

```diff
@@ -0,0 +1,17 @@
+-# Partial: used polymorphically for building and unit attributes
+-#   Arguments:
+-#     housingable: currently a building or unit object
+-#     attribute: a HousingAttribute object
+
+= content_for :modal_title do
+  Edit Attribute
+
+= simple_form_for @attribute, url:  polymorphic_path([@housingable, @attribute]) do |f|
+  = f.input :name, disabled: true
+  = f.input :value, collection: @attribute.existing_values, input_html: { class: :select2, data: { tags: true }}, include_blank: true, hint: 'If you don\'t see the value required, type it in and select it'
```

Comment:
I believe if you use `as: :select2` then you don't have to add the class or the javascript at the bottom of the partial.

Reply:
Yes, thanks for that.